### PR TITLE
Add fixed workflows

### PR DESCRIPTION
**Pull request by wesrice**

### CODE REVIEW

File: .github/workflows/production.yml [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      - id: node-modules
+        run: echo "::set-output name=hash::${{ hashFiles('**/yarn.lock') }}"
+
+  build:
+    needs: cache-keys
+    name: Build
+    runs-on: ubuntu-latest
+
+    outputs:
+      build-cache-hit: ${{ steps.cache-node-modules.outputs.cache-hit }}
```
Comment (kevcenteno) [inline .github/workflows/production.yml:37]: Shouldn't this be set to `steps.cache-build.outputs.cache-hit`?

File: .github/workflows/production.yml [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    name: Test – Accessibility
+    needs: [cache-keys, build]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+
+      - name: Get cached dependencies
+        id: cache-node-modules
```
Comment (kevcenteno) [inline .github/workflows/production.yml:85]: This isn't used anywhere

File: .github/workflows/production.yml [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    outputs:
+      build: build-${{ steps.build.outputs.hash }}-v1
+      node-modules: node-modules-${{ steps.node-modules.outputs.hash }}-v1
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+
+      - id: build
+        run: echo "::set-output name=hash::${{ hashFiles('content/**', 'firebase/**', 'functions/src/**', 'src/**', 'static/**', '**/package.json', '**/yarn.lock') }}"
```
Comment (kevcenteno) [inline .github/workflows/production.yml:26]: This should include `bin/**`, `sitemap-query-string.js` should be moved there, and the script in `package.json` that runs it should be updated

File: .github/workflows/production.yml [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+    name: Test – Unit
+    needs: [cache-keys, build]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+
+      - name: Get cached dependencies
+        id: cache-node-modules
```
Comment (kevcenteno) [inline .github/workflows/production.yml:107]: This isn't used anywhere

File: .github/workflows/production.yml [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+
+      - name: Set up Google Cloud SDK
+        uses: google-github-actions/setup-gcloud@daadedc81d5f9d3c06d2c92f49202a3cc2b919ba # v0.2.1
+        with:
+          project_id: ${{ env.FIREBASE_PROJECT_ID }}
+          service_account_key: "${{ env.FIREBASE_SERVICE_ACCOUNT }}"
+          export_default_credentials: true
+
+      - name: Get cached dependencies
+        id: cache-node-modules
```
Comment (kevcenteno) [inline .github/workflows/production.yml:186]: This isn't used anywhere

File: .github/workflows/production.yml [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+      (needs.test-a11y.result == 'success' || needs.test-a11y.result == 'skipped') &&
+      (needs.test-unit.result == 'success' || needs.test-unit.result == 'skipped')
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+
+      - name: Get cached build
+        id: cache-build
```
Comment (kevcenteno) [inline .github/workflows/production.yml:169]: This isn't used anywhere

**Review by kevcenteno — CHANGES_REQUESTED** [review]
A few changes are needed.  The comments on the production file also applies to the staging file.

File: .github/workflows/production.yml (hunk 1 above)
Comment (wesrice) [reply .github/workflows/production.yml:37]: This output is capturing if the `cache-node-modules` step of the current job used a previous build or not. Perhaps the name of the output could be better? Does the code provide more clarity?

```yml
outputs:
      is-previously-tested: ${{ steps.cache-node-modules.outputs.cache-hit }}
```

and subsequently

```yml
test-a11y:
    if: needs.build.outputs.is-previously-tested != 'true'
```

Comment (kevcenteno) [reply .github/workflows/production.yml:37]: Why test if the node modules cache was hit instead of the build cache?

Comment (wesrice) [reply .github/workflows/production.yml]: Ah, I see what you're saying now. I was crossed-up and thinking of the `cache-keys` outputs for some reason. Updating this.

**Review by kevcenteno — APPROVED** [review]

**Pull request merged**